dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.unversioned.servlet-0.0.feature

```diff
@@ -0,0 +1,8 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.unversioned.servlet-0.0
```

Comment:
These should all be io.openliberty prefixed, not com.ibm.websphere.appserver prefixed.  All new features should be io.openliberty prefixed.